components/nav.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="header">
+    <div class="title">
+      <a class="reglink" href="/">
+      OFS Farms
+      </a>
+    </div>
+    <div class="hlinks">
+      <ul>
+        <li class="hlink-item">
+          <NuxtLink to="/Shop" class="reglink">
```

Comment:
My bad! Didn't notice but usually links are kebab-case. 
```suggestion
          <NuxtLink to="/shop" class="reglink">
```